src/Functions/modulo.cpp

```diff
@@ -1,10 +1,7 @@
 #include <Functions/FunctionFactory.h>
 #include <Functions/FunctionBinaryArithmetic.h>
 
-#if defined(__SSE2__)
-#    define LIBDIVIDE_SSE2 1
-#endif
-
+#include <libdivide-config.h>
```

Comment:
@azat please, take a look if it's fine. Maybe, makes sense to tune the contrib/libdivide-cmake/libdivide-config.h?

Reply:
Yep, this should be compatible.